Source/ModelGroups/Compatibility.cs

```diff
@@ -103,183 +124,127 @@ void SetModelShouldLoad(List<Property> properties, string loadableStatus = ":whi
                 properties.Add(new Property(PropertyName.ModelShouldLoad, loadableStatus));
             }
 
-            glTFLoader.Schema.Gltf SetPostRuntimeAtRoot(glTFLoader.Schema.Gltf gltf)
+            void SetPostRuntimeAtRoot(glTFLoader.Schema.Gltf gltf)
             {
                 // Add an simulated feature at the root level
-                gltf = new ExperimentalGltf1(gltf)
+                var experimentalGltf = (ExperimentalGltf)gltf;
+                experimentalGltf.Lights = new[]
                 {
-                    lights = new ExperimentalGltf1.Light
+                    new ExperimentalLight
                     {
                         Color = new float[] { 0.3f, 0.4f, 0.5f }
                     }
                 };
-
-                return gltf;
             }
 
-            glTFLoader.Schema.Gltf SetPostRuntimeInProperty(glTFLoader.Schema.Gltf gltf)
+            void SetPostRuntimeInProperty(glTFLoader.Schema.Gltf gltf)
             {
                 // Add an simulated feature into an existing property
-                gltf.Nodes[0] = new ExperimentalGltf1.Node(gltf.Nodes[0])
-                {
-                    Light = 0.5f
-                };
-
-                return gltf;
+                var experimentalNode = (ExperimentalNode)gltf.Nodes[0];
+                experimentalNode.Light = 0;
```

Comment:
Was there a reason to change this value from 0.5f to 0?

Reply:
`0.5f` didn't make sense. It should be an index to the lights array.

Reply:
When I wrote the value as 0.5f I had thought of it as something like a global multiplier for lighting on the object. Using it as an index doesn't make any more sense to me, since there isn't an array for it to index to.

That said, I'm not going to push very hard on this once since the value doesn't do anything either way.

Reply:
I see what you mean. If it were a factor of some kind, we probably should call it with the word `factor` in it, but yeah, it doesn't really matter.